uui-components/src/overlays/Dropdown.tsx

```diff
@@ -41,7 +40,7 @@ export interface DropdownProps extends Partial<IEditable<boolean>> {
     closeOnMouseLeave?: 'toggler' | 'boundary' | false;
 
     portalTarget?: HTMLElement;
-    boundaryElement?: PopperJS.Boundary | Element;
+    boundaryElement?: Boundary;
```

Comment:
In Popper 2 'Element' can't be passed?

Reply:
![image](https://user-images.githubusercontent.com/33173033/110933538-f31deb00-833d-11eb-87f2-4aac8d17fdb1.png)
Boundary is typed as HTMLElement, which inherits from Element